FESS-GUI/setpassworddialog.cpp

```diff
@@ -13,6 +14,7 @@ SetPasswordDialog::SetPasswordDialog(QWidget *parent) :
     ui(new Ui::SetPasswordDialog)
 {
     ui->setupUi(this);
+    qsrand(time(NULL));
```

Comment:
No problem, we can move it back. I was cleaning up libraries and it compiled and ran fine on my computer so I had not clue ctime was needed.